src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -73,22 +77,26 @@ Each issue preference option has its own advantages. You can select any preferen
 
       <td>
         Only one issue will be open at a time for the entire policy.
+      </td>
 
+      <td>
         * Fewest number of notifications
         * Requires immediate action and closing the issues to be effective
       </td>
     </tr>
 
     <tr>
       <td>
-        [One issue per condition](#preference-condition)
+        [One issue per condition and signal](#preference-condition)
       </td>
 
       <td>
-        One issue will be open at a time for each condition in your policy.
+        An issue will be created for every signal on every condition in your policy.
```

Comment:
This is incorrect. For the "per condition" setting, one issue per condition is how that works.